gravitee-apim-console-webui/src/management/api/endpoints-v4/endpoint-group/api-endpoint-group.component.html

```diff
@@ -39,6 +39,16 @@
           ></api-endpoint-group-configuration>
         </div>
       </mat-tab>
+      <mat-tab label="Health-check" *ngIf="healthCheckForm && endpointGroup.type != 'mock'">
```

Comment:
as discussed with you we should not display this page when we are working on a message or tcp API. Can we change this condition and add some unit tests to be sure this rule is respected ?

Reply:
good shout, fixed and tests added